CHANGES.md

```diff
@@ -1,3 +1,31 @@
+### Unreleased
+
+#### Fixed
+- **irmin-pack**
+  - Fixed a bug in `inode`, irmin-pack's tree-like data-structure speeding up
+    operations on large directories. Under some circumstances, two directories
+    containing the same entries were hashed differently after deleting an entry.
+
+	For directories containing at most `Conf.entries` (32) entries, no bugs
+	occurred. In this case, this change does not induce any user-facing change.
+
+	For directories containing at most `Conf.stable_hash` (256) entries, only a
+	few hashes internal to `inode` could have been inconsistent, but not the
+	directory's hash.
```

Comment:
users will just see the root hashes, so it's not an issue if the internal hashes are not deterministic. So I suggest something like this:

```
For directories containing at most `Conf.stable_hash` entries (by default 256), no bug occurred. In this case, this change does not induce any user-facing change.

For directories containing more than `Conf.stable_hash` entries, the directory's hash might depends on the past sequence of operations and is thus not deterministic.
```

and remove the next paragrapg,

Reply:
As Ioana suggested, you can keep the details in the commit message (as the next developper doing `git blame` will like to know about the gory details).

Reply:
Okey, i'll do exactly that. Thank you